framework/Samplers/Sampler.py

```diff
@@ -900,6 +999,13 @@ def pcaTransform(self,varsDict,dist):
     manifestVariablesDict = dict(zip(varsDict['manifestVariables'],manifestVariablesValues))
     self.values.update(manifestVariablesDict)
 
+  def _checkSample(self):
+    """
+      Checks the current sample for consistency with expected contents.
```

Comment:
what is supposed to do?

Reply:
It checks the sample, assuring that it provides the expected components. Chiefly used by the AdaptiveSampling base class but added here so as to be consistent across all Samplers.